### PR TITLE
Timezone List pattern data change

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

File: g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/timezone/CldrTimeZoneUtils.java [hunk 1]
```diff
@@ -57,25 +60,37 @@ public static String createTimeZoneNameJson(JSONObject metaZonesJson, JSONObject
                 e.printStackTrace();
             }
 
-            TimeZoneDisplayName name = new TimeZoneDisplayName();
+            Map<String, TimeZoneDisplayName> metaZonePerperties = new TreeMap<String, TimeZoneDisplayName>();
+           
             if (metazoneValue == null) {
+                TimeZoneDisplayName longPerp = new TimeZoneDisplayName();
                 String stard = MessageFormat.format(regionFormatTypeStandard, exemplarCity);
```
Comment (Xiaochao8) [inline g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/timezone/CldrTimeZoneUtils.java:67]:
```suggestion
                String standard = MessageFormat.format(regionFormatTypeStandard, exemplarCity);
```

**Review by Xiaochao8 — COMMENTED** [review]
What's the issue for this PR?

File: g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/timezone/CldrTimeZoneUtils.java (hunk 1 above)
Comment (tigershi) [reply g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/timezone/CldrTimeZoneUtils.java]: done

**Comment by tigershi** [comment]
> What's the issue for this P

**Review by chaoq0318 — DISMISSED** [review]

**Review by Xiaochao8 — APPROVED** [review]

**Review by chaoq0318 — APPROVED** [review]

**Comment by nannany00** [comment]
Please check the bug description again, there're still some items not changed. Thanks.

**Pull request merged**